src/metabase/api/dashboard.clj

```diff
@@ -924,6 +925,69 @@
       (create-editable-table-card! dashboard-id dashcard table-id)
       dashcard)))
 
+(defn- init-grid-actions
```

Comment:
should this lives somewhere other than dashboard api? maybe dashcard model, or even in action?

Reply:
Right now i see this hack living in the API, but we can revisit this when we're cleaning things up for the merge into master.

The post-hack version of this probably involves creating first class Action rows instead of more nested visualization settings, and then the code will definitely move to the model namespace.